### PR TITLE
Support OS selection

### PR DESCRIPTION
**Pull request by pkliczewski**
Due to the issues with understanding os type for imported vm we want
to expose the selection to the user. This fix is considered to be temporaty
and once we will find how to get correct os this code will be removed

### CODE REVIEW

File: bin/job [hunk 1]
```diff
@@ -81,9 +83,12 @@ transformVM() {
   BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
 
   # Create job offlinevirtualmachine
+  if [[ "$SRCOS" = "Linux" ]]; then
 tee $VMYAML <<EOY
 apiVersion: kubevirt.io/v1alpha1
 kind: OfflineVirtualMachine
+labels:
```
Comment (fabiand) [inline bin/job]: labels are a child of metadata.

File: bin/job [hunk 2]
```diff
@@ -106,6 +111,63 @@ spec:
...
+          present: false
+        pit:
+          tickPolicy: delay
+        rtc:
+          tickPolicy: catchup
+        hyperv: {}
+---
+apiVersion: kubevirt.io/v1alpha1
+kind: OfflineVirtualMachine
+labels:
```
Comment (fabiand) [inline bin/job]: labels are a child of metadata.

**Review by fabiand — CHANGES_REQUESTED** [review]

File: bin/job (hunk 1 above)
Comment (pkliczewski) [reply bin/job]: Will fix, thanks

File: bin/job (hunk 2 above)
Comment (pkliczewski) [reply bin/job]: Will fix, thanks

**Comment by pkliczewski** [comment]
@fabiand please review again

**Pull request closed without merging**